src/macros.rs

```diff
@@ -372,6 +372,28 @@ macro_rules! crate_version {
     };
 }
 
+/// Allows you pull the authors for the app from your Cargo.toml at compile time as 
+/// "author1 lastname. <author1@gmail.com>", "author2 lastname. <author1@gmail.com>"
```

Comment:
I think using the example domain (@example.org or @example.com) would be better. Also, the second email should not equal the first email.